frontend/src/tests/user-permission-tests.js

```diff
@@ -14,42 +14,142 @@ import { databaseSeeder } from "./setup";
 export function userPermissionsTests(api) {
     const { apiGET, apiPOST } = api;
 
-    const newUserData = {
-        utorid: "userxx",
+    let defaultUser;
+
+    const instructorOnlyUserData = {
+        utorid: "user_permission_test_instructor_only_utorid",
         roles: ["instructor"],
     };
 
+    const TAOnlyUserData = {
```

Comment:
lower case `ta`. There's no need for the word `Data` to be in this variable name.